v3.2 release notes (#1162)
ezsystems/developer-documentation: docs/releases/ibexa_dxp_v3.2.md

# Ibexa DXP v3.1

**Version number**: v3.2

**Release date**: October 20, 2020

**Release type**: Fast Track

## Notable changes

![New login page]()

### New UI

This version offers a completely reworked user interface, covering all of the Back Office,
including eCommerce administration.

![New Content structure]()

![Commerce administration]()

### DAM connector

### Autosave

> eZ Platform can now save your edits in a Content item or product automatically to help you preserve the progress in an event of a failure. For more information, see [Autosave](../publishing/publishing.md#autosave).

Reviewer: ```suggestion
eZ Platform can now save your edits in a Content item or product automatically to help you preserve the progress in an event of a failure. For more information, see [Autosave](https://doc.ezplatform.com/projects/userguide/en/latest/publishing/publishing/#autosave).
```

> # Ibexa DXP v3.1

Reviewer 2: ```suggestion
# Ibexa DXP v3.2
```

### Autosave

> eZ Platform can now save your edits in a Content item or product automatically to help you preserve the progress in an event of a failure. For more information, see [Autosave](../publishing/publishing.md#autosave).

Author: Link to user guide, needs a full address.

For more information, see [Calendar widget](https://doc.ibexa.co/projects/userguide/en/latest/publishing/advanced_publishing_options/#calendar-widget).

> ### Cloning content types

Author: ```suggestion
### Cloning Content Types
```
